src/components/Table/SimpleTable.js

```diff
@@ -55,12 +59,21 @@ class SimpleTable extends React.Component {
     renderDetails: null,
     columns: null,
     onSortChanged: () => null,
+    getId: (item) => {
```

Comment:
This is concerning as a function in the shared-components library. Shared components should not assume the shape of `item`. I think this should be a `noop` function  ( `getId: () => null`) instead. This would place the responsibility of identifying the id on the calling application.

Reply:
I suggested it to provide some sane out-of-the-box behavior since this component is focused on boostrapping a table from some data without a lot of effort. Without some default implementation, you wouldn't be able to expand details of a row at all. I don't really think it's problematic to take a guess at something useful and fallback to comparing the strict values of the items with stringify.

Reply:
Ok, I'm willing to concede that some sort of default may seem helpful from a usage perspective. I'm still a little concerned about this within the shared-components library but not enough to reject it.

Reply:
I think this component is a bit of an oddball. We could perhaps spin off a scl-recipes library that combines components into common bootstrapped patterns maybe.